packages/plugin-e2e/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "outDir": "./dist",
-    "declaration": true
+    "declaration": true,
+    "stripInternal": true
```

Comment:
Would like some types to be internal so added this option. Maybe this should go in the base config? Thoughts @jackw?

Reply:
Just out of curiosity - how do we decide which types should be private and which ones should be public? I sometimes find it useful during development to have access to as many types as possible when working with a package, while on the other hand it can tie our hands if we expose too much, as we can hardly change what a lot of people depend on.

Reply:
Good question. Think I just don't want to overload consumer with irrelevant types etc. But may be better to save this for later.